tajo-core/src/main/java/org/apache/tajo/querymaster/QueryMasterManagerService.java

```diff
@@ -168,7 +170,7 @@ public void statusUpdate(RpcController controller, TajoWorkerProtocol.TaskStatus
       done.run(TajoWorker.TRUE_PROTO);
```

Comment:
I mean, we can assume that the remote call is successfully finished if the InvocationFailure does not occur.